Modules/IO/NIFTI/test/itkNiftiImageIOTest12.cxx

```diff
@@ -69,30 +78,59 @@ itkNiftiImageIOTest12(int ac, char * av[])
   }
 
   using Hasher = itk::Testing::HashImageFilter<ImageType>;
-  Hasher::Pointer hasher = Hasher::New();
-  hasher->SetInput(image);
-  hasher->InPlaceOff();
-  hasher->Update();
-
-  std::string originalHash = hasher->GetHash();
 
+  const std::string originalHash = [&]() -> std::string {
```

Comment:
Why not name this lambda and reuse it on line 97?

Reply:
Good point.  Done.